autogen/agentchat/contrib/critic_agent.py

```diff
@@ -0,0 +1,55 @@
+from typing import Callable, Dict, Literal, Optional, Union
+
+from autogen.agentchat.conversable_agent import ConversableAgent
+from autogen.runtime_logging import log_new_agent, logging_enabled
+
+
+class CriticAgent(ConversableAgent):
```

Comment:
Lint question: do you think it is better to combine the critic agent, quantifier agent, and subcritic agent into one file, such as `eval_agents.py`?